lib/web_ui/test/text_editing_test.dart

```diff
@@ -110,14 +102,23 @@ void main() {
       expect(document.activeElement, document.body);
     });
 
+    test('Respects read-only config', () {
+      final InputConfiguration config = InputConfiguration(readOnly: true);
+      editingElement.enable(
+        config,
+        onChange: trackEditingState,
+        onAction: trackInputAction,
+      );
+      expect(document.getElementsByTagName('input'), hasLength(1));
+      final InputElement input = document.getElementsByTagName('input')[0];
+      expect(editingElement.domElement, input);
+      expect(input.getAttribute('readonly'), 'readonly');
+
+      editingElement.disable();
```

Comment:
Should we `expect` something after this method call?

Reply:
This is just cleanup after the test. There are a few other tests in this file that check the behavior of `TextEditingElement.disable()`.